sql/PostLinks_pre.sql

```diff
@@ -2,7 +2,8 @@ DROP TABLE IF EXISTS PostLinks CASCADE;
 CREATE TABLE PostLinks (
    Id                int         PRIMARY KEY ,
    CreationDate      timestamp   not NUll    ,
-   PostId            int         not NULL    ,
-   RelatedPostId     int         not NULL    ,
-   LinkTypeId        int         not Null
+   PostId            int         , -- not NULL    ,
```

Comment:
Is the removal of the not-NULL constraint artifacts from the PR about the Foreign keys? Is it needed for adding JSON fields?